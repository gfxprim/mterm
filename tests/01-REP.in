10 2
a\e[8b
