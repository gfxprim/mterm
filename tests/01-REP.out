 ----------
|aaaaaaaaa |
|          |
 ----------
size 2x10 cursor 0x9
